src/Book/EditBook/EditBook.tsx

```diff
@@ -16,6 +19,14 @@ const EditBook = (props: Props) => {
     const [openNewCopy, setOpenNewCopy] = useState<boolean>(false);
     const [newCopyError, setNewCopyError] = useState<boolean>(false);
 
+    const [actDeactCopyError, setActDeactCopyError] = useState<boolean>(false);
+
+    const [openActDeactCopy, setOpenActDeactCopy] = useState<boolean>(false);
```

Comment:
I leave the comment here but applies in general.

Try to name variables, components, functions, etc with a complete name. Is better to have a long name than a name that could be misunderstood. 

In this case is preferable the name `openActivateDeactivateCopyModal`.